src/fr/composants/bouton/cas-dutilisation.md

```diff
@@ -67,55 +66,70 @@ Utilisez le type « Réinitialisation » lorsque vous vous attendez à ce qu'un
 
 Utilisez le type « Lien » lorsque vous vous attendez à ce qu'une personne :
 
-- se rendre vers d'autres pages ou vers des sites externes à l'aide d'un bouton de type « lien ».
+- Se dirige vers d'autres pages ou sites externes.
 
-Remarque : Utilisez le [composant lien]({{ links.link }}) lorsque vous devez insérer un lien dans le texte d'un paragraphe ou dans le corp du texte.
+Remarque : Utilisez le [composant lien]({{ links.link }}) pour intégrer un lien dans le texte d'un paragraphe ou d'une barre de navigation.
 
 ## Choisissez le rôle du bouton
 
-Un rôle est un sous-type de bouton à usage spécifique sur une page.
+Le rôle du bouton change son apparence afin de signaler visuellement l'action qui se produira. Choisissez le rôle qui correspond le mieux à l'action.
 
 <div class="remove-empty-p">
 <gcds-grid columns="1fr" columns-tablet="1fr 2fr" align-items="start">
-  {% componentPreview "Aperçu du bouton principal" "px-225 py-300" "" %}
+  {% componentPreview "Aperçu du bouton principal" "px-225 py-300" "mt-500" %}
   <gcds-button button-role="primary">Soumettre</gcds-button>
   {% endcomponentPreview %}
   <div>
-    <h3 class="mt-0">Utilisations du bouton principal</h3>
+    <h3 class="mt-0">Bouton principal</h3>
     <ul class="list-disc mb-300">
-      <li>L'action la plus importante d'une page.</li>
       <li>Actions cruciales au sein d'un processus ou en tant que bouton par défaut.</li>
-      <li>Limitation à une seule utilisation.</li>
+      <li>Utiliser une seule fois par page.</li>
+    </ul>
+    <p>Conseil : La présence de plus d'un bouton principal d'action sur une même page fait en sorte qu'il est plus difficile pour une personne de savoir ce qu'elle doit faire ensuite.</p>
+  </div>
+</gcds-grid>
+<br/>
+<gcds-grid columns="1fr" columns-tablet="1fr 2fr" align-items="start">
+  {% componentPreview "Aperçu du Bouton de démarrage" "px-225 py-300" "mt-500" %}
+  <gcds-button button-role="start">Bouton de démarrage </gcds-button>
+  {% endcomponentPreview %}
+  <div>
+    <h3 class="mt-0">Bouton principal</h3>
+    <p><i>Anciennement le « bouton de super-tâche ».</i></p>
```

Comment:
Same comment here: Can we switch this back to a n`<em>`? The `<i>` tag is not accessible because it only visually emphasizes text.